### PR TITLE
Issue 17 feat/UI prepare text component

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

**Review by Veryyapeee — APPROVED** [review]

File: src/components/atoms/Text/Text.jsx [hunk 1]
```diff
@@ -2,15 +2,17 @@ import PropTypes from 'prop-types';
...
-export const Text = ({ children, variant }) => {
-    return <p className={`atom__text atom__text__${variant}`}>{children}</p>;
+export const Text = ({ children, type, size }) => {
+    return <span className={`atom__text atom__text${type} atom__text${size}`}>{children}</span>;
 };
 
 Text.propTypes = {
-    children: PropTypes.node,
-    variant: PropTypes.oneOf(['primary', 'secondary'])
+    children: PropTypes.string,
```
Comment (T0ffik) [inline src/components/atoms/Text/Text.jsx:10]: Here shouldn't be .isRequired? @Veryyapeee?

Comment (Veryyapeee) [reply src/components/atoms/Text/Text.jsx:10]: good catch, it should be, we don't want to render empty component

Comment (Veryyapeee) [reply src/components/atoms/Text/Text.jsx:10]: @Mariusz-web correct that pls

Comment (Mariusz-web) [reply src/components/atoms/Text/Text.jsx]: done, i thing so

**Review by Veryyapeee — APPROVED** [review]

**Review by T0ffik — APPROVED** [review]

**Pull request merged**